webpack.config.js

```diff
@@ -13,8 +13,5 @@ module.exports = {
       loader: 'babel-loader'
     }]
   },
-  resolve: {
-    extensions: ['', '.js']
-  },
```

Comment:
It was required in Webpack 1 not in Webpack 2. https://webpack.js.org/guides/migrating/#resolve-extensions